sdk/core/src/entities/sboms/sbom.rs

```diff
@@ -132,7 +132,7 @@ impl Sbom {
         let dependency_refs = package.dependency_refs.clone();
 
         let created = match platform::time::iso8601_timestamp() {
-            Ok(created) => Some(created),
+            Ok(created) => created,
```

Comment:
Got it, will address.

Reply:
So, after reviewing the code, I feel it makes more sense to return a String and make an Option out of that string later.  The functions that _was returning_ an option, iso8601_timestamp(), no longer does:

```
/// Generates an rfc3339 time stamp which is an implementation of
/// a iso8601 timestamp.
pub fn iso8601_timestamp() -> Result<String, Error> {

    // Get the unix timestamp
    let unix_timestamp: u64 = timestamp()?;

    // if the u64 is greater than the maximum value an i64 can hold
    // (which is 9223372036854775807), it will wrap around and
    // become a negative number. A unix timestamp will never be
    // this large however, so a simple test should make sure that
    // we get a reasonable value for the timestamp.
    let i64_unix_ts = unix_timestamp as i64;
    if i64_unix_ts <= 0 {
        return Err(Error::Time(format!(
            "Unix timestamp is too large: {}",
            unix_timestamp
        )));
    }

    let naive_datetime = NaiveDateTime::from_timestamp_opt(i64_unix_ts, 0).unwrap();
    let datetime: DateTime<Utc> = DateTime::from_utc(naive_datetime, Utc);

    // ISO 8601 allows some freedom over the syntax and RFC 3339
    // exercises that freedom to rigidly define a fixed format.
    Ok(datetime.to_rfc3339())
}
```

And the two places using it in the code, now create an option out of it as soon as they have the value:
```
       let created = match platform::time::iso8601_timestamp() {
            Ok(created) => Some(created),
            Err(e) => { 
                println!("timestamp creation failed with {}", e);
                None
            }
        };
```
And
```
        // Ensure the display timestamp is set if successful.
        sbom.created = Some(platform::time::iso8601_timestamp()?);
```